utils/generate_types/overrides-test.d.ts

```diff
@@ -91,7 +91,7 @@ export interface FullConfig<TestArgs = {}, WorkerArgs = {}> {
   shard: { total: number, current: number } | null;
   updateSnapshots: 'all' | 'none' | 'missing';
   workers: number;
-  webServer: TestConfigWebServer | null;
+  webServer: TestConfigWebServer | TestConfigWebServer[] | null;
```

Comment:
Unfortunately (or fortunately), TS forces this (lest we refactor much more), since `FullConfigInternal extends FullConfigPublic` so `webServer` type cannot be `webServer: TestConfigWebServer | null;` in the original and then `webServer: TestConfigWebServer | TestConfigWebServer[] | null;` on the extension.

If we add a `launchConfigs` config in the future, it will only exist on `FullConfigInternal` as `_launchConfigs`.

Reply:
Discussed offline. This is now updated.